examples/dem/3d-rectangular-hopper/Lethe_pyvista_tools.py

```diff
@@ -0,0 +1,125 @@
+#############################################################################
+"""Class of methods to post-process lethe results with pyvista"""
+#############################################################################
+
+#Import modules
```

Comment:
If you are using the Lethe_py_vista tools and we start using it in multiple places, then I think it would be a good idea to add it to contrib/postprocessing and maintain it there instead of copying it all over the place. What do you think @voferreira ?

Reply:
Sure! It can be done within this PR or I can add it with some documentation in another one. It is up to you

Reply:
It needs a bit of cleaning. I will do it today.

Reply:
I think you're going to be good, but tell me if you need help with the clean up.